server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -257,10 +256,20 @@ private static boolean createDirs(VolumeManager fs, InstanceId instanceId, Set<S
         fs.mkdirs(
             new Path(new Path(baseDir, Constants.VERSION_DIR), "" + AccumuloDataVersion.get()),
             new FsPermission("700"));
+        log.info("Created directory {}", baseDir);
```

Comment:
I don't see any javadoc in HDFS for why the underlying `FileSystem.mkdirs` call would return `false`. The result of the `fs.mkdirs` call above and below is unchecked. We may want to do something like:
```
boolean success = fs.mkdirs ...
if (!success && !fs.exists(Path)) {
  throw new Something;
} else {
  log.info("Created directory {}", baseDir);
}
```

Could use the same pattern for the iidPath file below.

Reply:
Addressed in 152388d51b